### PR TITLE
feat: ensures zero fee address is correctly set

### PR DESCRIPTION
**Pull request by ckartik**
```
.primev % cast balance 0xADB4257612d45f12570533308b20AC77DBfeB85a
50000000000000000000
.primev % cast send --from 0xADB4257612d45f12570533308b20AC77DBfeB85a --private-key cf194d7afcff3cf4245650389232b58ea13d6bbe744463d88e66ce45ce6c0034 0x0000000000000000000000000000000000000000 --value 1

blockHash               0x8d3b34a53fbd5cdb0f4a5e1ecc185c8261fe6f7e118887c27130418d006cb1fa
blockNumber             2
contractAddress
cumulativeGasUsed       21000
effectiveGasPrice       766024068
from                    0xADB4257612d45f12570533308b20AC77DBfeB85a
gasUsed                 21000
logs                    []
logsBloom               0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
root
status                  1 (success)
transactionHash         0x10edf9347c1cb9cd0b4376cf5e6ad3e46d4543c3a3701959d50005c04bd4f0f2
transactionIndex        0
type                    2
blobGasPrice
blobGasUsed
authorizationList
to                      0x0000000000000000000000000000000000000000
.primev % cast balance 0xADB4257612d45f12570533308b20AC77DBfeB85a

49999999999999999999
```

Observe above that the balance is only decremented 1 wei on a transfer of 1 wei, meaning the zero-fee-address paid zero in transaction fees.

### CODE REVIEW

File: geth-poa/genesis.json [hunk 1]
```diff
@@ -1,74 +1,32 @@
 {
-    "config": {
```
Comment (aloknerurkar) [inline geth-poa/genesis.json:2]: Why is this changed?

**Review by aloknerurkar — APPROVED** [review]

**Review by mrekucci — APPROVED** [review]
